examples/linux-policyfile-cookbook/README.md

```diff
@@ -0,0 +1,47 @@
+# linux-policyfile-cookbook
+
+This cookbook meant to serve as an example of testing a Policyfile cookbook via
```

Comment:
Can we fix some of this 80 character line formatting since this isn't source code?

Reply:
I strongly prefer the formatting for the sake of PR reviews. :(

To me it is code and let the renderer make it look pretty.

Reply:
Wish we had a company style guide for Markdown...but I don't know of any.

Reply:
Yeah, let's just go with this.